lib/bookingsync_portal.rb

```diff
@@ -21,6 +21,13 @@ module BookingsyncPortal
   mattr_accessor :portal_name
   @@portal_name = 'Portal'
 
+  # source name for use in bookings
+  mattr_accessor :source_name
+
+  # Allow to create a remote rental from this app.
+  mattr_accessor :create_remote_rental
+  @@create_remote_rental = false
```

Comment:
Not directly related to this PR, but why do we actually use class variables instead of class instance variables?